garminconnect/__init__.py

```diff
@@ -1022,13 +1039,13 @@ def get_activity_hr_in_timezones(self, activity_id):
 
         return self.connectapi(url)
 
-    def get_activity_evaluation(self, activity_id):
-        """Return activity self evaluation details."""
+    def get_activity(self, activity_id):
```

Comment:
Possibly it makes sense to leave get_activity_evaluation as is, and create another (identical) method called get_activity, but to me, that feels less good.  

Maybe it makes sense to add a warning if someone has been calling get_activity_evaluation -- leave it as a function but have it call get_activity instead? I'm open to suggestions, but I figured I'd create the PR to get the ball rolling